example/exampleUser/ExampleUserProvider.js

```diff
@@ -141,7 +141,7 @@ export default class ExampleUserProvider extends EventEmitter {
   }
 
   getPossibleRoles() {
-    return this.user.getRoles();
+    return this.loginPromise.then(() => this.user.getRoles());
```

Comment:
Normal practise, is to use underscore (_) as a parameter in the promise constructor. Just a nitpick

Reply:
Given that the argument is unused